packages/dashboard/src/portal/views/Farms.vue

```diff
@@ -283,13 +292,26 @@ export default class FarmsView extends Vue {
   count = 0;
   farmsIds: any;
   initLoading = false;
+  errorMessage = "";
 
-  // Life hooks
-  async mounted() {
+  async getFarms() {
     // not logged in? login: get farms, nodes
     if (this.$api && this.$store.state.credentials) {
-      this.farms = await getFarm(this.$api, this.$store.state.credentials.twin.id);
-      this.loadingFarms = false;
+      this.loadingFarms = true;
+      this.errorMessage = "";
+      try {
+        this.farms = await getFarm(this.$api, this.$store.state.credentials.twin.id);
+      } catch (error: any) {
+        if (error instanceof Error) {
+          this.errorMessage = `An error occurred during the attempt to fetch farms due to a ${error.message.toLocaleLowerCase()}`;
+          console.log(
+            "An error occurred during the attempt to fetch farms due to a ",
+            error.message.toLocaleLowerCase(),
+          );
```

Comment:
Everything looks great but can we remove the log here? Error has already been shown to the user so no need to log it.

Reply:
Please take a look again